src/H5LS.c

```diff
@@ -187,7 +187,12 @@ herr_t readLSConf(char *fname, cache_storage_t *LS) {
       if (get_replacement_policy_from_str(mac) > 0)
         LS->replacement_policy = get_replacement_policy_from_str(mac);
     } else {
-      LOG_WARN(-1, "Unknown configuration setup:", ip);
+      char temp_ip[256];
+      strncpy(temp_ip, ip, sizeof(temp_ip) - 1);
+      temp_ip[sizeof(temp_ip) - 1] = '\0';
```

Comment:
Seems fine, though is there a reason not to just directly print it into `error_msg` with snprintf?

Reply:
fixed